K2Bridge/Models/Request/Aggregations/Metric/Max/MaxAggregation.cs

```diff
@@ -11,7 +11,6 @@ namespace K2Bridge.Models.Request.Aggregations
     /// <summary>
     /// A single-value metrics aggregation that computes the max of numeric values that are extracted from the aggregated documents.
```

Comment:
Is the 'max' directory needed? same for the rest (avg etc)

Reply:
I need to admit this is a personal preference to access code quickly. And will structure better the code if a Bucket/Metric need several files (at the moment this is only the case for Terms). @nebrass @tomconte -> Your opinion ?

Reply:
Looks good to me, but then we should probably use the same kind of structure for the other parts of the code.

Reply:
Yes, that's the plan for the answer.

Reply:
so do we add a user story to align the rest of the code?